app.py

```diff
@@ -13,23 +14,29 @@
 import api.session_items as session
 import api.sorter as sorter
 from models.role import Role
-from models.user import User
 from models.view_model import ViewModel
 
 def create_app():
     app = Flask(__name__)
     config = AppConfig()
-    data_manager = MongoHelper(config)
+    data_manager = MongoHelper(config, app.logger)
     app.secret_key = config.SECRET_KEY
     app.data_manager = data_manager
     app.config.from_object(config)
     auth_config = AuthConfig()
+    app.logger.setLevel(config.LOG_LEVEL)
```

Comment:
Minor, but worth noting that you set this here _after_ the MongoHelper has already done some logging at the Debug/Info level - it will affect only a small proportion of your logs, but it can confuse (it did for me at least!)